packages/breadcrumbs/index.js

```diff
@@ -1,20 +1,25 @@
-import { FabricWebComponent } from '../utils';
+import { html } from 'lit';
+import { FabricElement } from '../utils';
 
-class FabricBreadcrumbs extends FabricWebComponent {
-  connectedCallback() {
-    const children = Array.from(this.children)
-      .map((child) => child.outerHTML)
-      .join('<span class="select-none" aria-hidden="true">/</span>');
+const separator = html`<span class="select-none" aria-hidden="true">/</span>`;
+const interleave = (arr) =>
+  [].concat(...arr.map((el) => [el, separator])).slice(0, -1);
 
-    this.shadowRoot.innerHTML += `<nav
-            aria-label="Her er du"
-            class="flex space-x-8"
-        >
-            <h2 class="sr-only">Her er du</h2>
-            ${children}
-        </nav>`;
+class FabricBreadcrumbs extends FabricElement {
+  connectedCallback() {
+    super.connectedCallback();
+    // Grab existing children at the point that the component is added to the page
+    // Interleave "/" separator with breadcrumbs
+    this._children = interleave(Array.from(this.children));
+  }
```

Comment:
intercept the child elements when the component is first added to the page and interleave them with slash character separators. Store the result as the internal property _children for use in the render method.

Reply:
This interleaving is reactive in Vue (and I think React), but not here. Are we OK with that? Should that be disclosed?

Reply:
Unsure. It is perhaps indicative of one of the differences between react/vue and custom elements. I toyed around with using dedicated child elements instead of this which is maybe one solution.

Reply:
When you say not reactive do you mean in the context of using this custom element in React or Vue, or as in if the user updates the props, it wouldn't update the DOM element? @pearofducks

Reply:
I mean this implementation in the custom element isn't watching children - so if the user adds/removes breadcrumbs it won't re-interleave them.